### PR TITLE
Create belin-agregation.csl

### PR DESCRIPTION
**Pull request by yanbabels**
I adapted the ENS de Lyon style to have different citation elements that correspond to what the publisher BELIN uses for its AGREGATION series

### CODE REVIEW

**Comment by yanbabels** [comment]
I just realized I added 'ENGLISH' but this style hasn't been localized yet and is in FRENCH

File: belin-agregation.csl [hunk 1]
```diff
@@ -0,0 +1,748 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" initialize="false" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Belin agrégation (english)</title>
+    <title-short>Belin</title-short>
+    <id>http://www.zotero.org/styles/belin-agregation</id>
+    <link href="http://www.zotero.org/styles/belin-agregation" rel="self"/>
+    <link href="http://cid.ens-lyon.fr/articles/zotero_style_cid.pdf" rel="documentation"/>
```
Comment (adam3smith) [inline belin-agregation.csl:9]: could you link to the documentation for Belin here?

File: belin-agregation.csl [hunk 2]
```diff
@@ -0,0 +1,748 @@
...
+    <author>
+      <name>Stéphane Marchand</name>
+      <email>stephane.marchand@ens-lyon.fr</email>
+    </author>
+    <author>
+      <name>Anatole Lucet</name>
+      <email>anatole.lucet@ens-lyon.fr</email>
+    </author>
+    <contributor>
+      <name>Yan Brailowsky</name>
```
Comment (adam3smith) [inline belin-agregation.csl:19]: Please make yourself the main author, even if you only changed a few things.

**Review by adam3smith — CHANGES_REQUESTED** [review]
A couple of small things -- also see damnation's comment above

**Comment by adam3smith** [comment]
@yanbabels please see the questions above

**Comment by adam3smith** [comment]
Closing due to inactivity -- we're happy to re-open this if you find time to come back to it: simply post a short message here.

**Comment by yanbabels** [comment]
hi,
basically, yes, although I did one minor change (putting idem instead of 
id). The problem is I don't know how to add the locale… I'm new to this, 
as you can see.
Is there a simple way to add the EN localization in the original file? I 
read the documentation online and I still don't get it.
Best
Yan

**Pull request closed without merging**